<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingPage1</class>
 <widget class="QDialog" name="ShoppingPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: bold 14px;
    color: white;</string>
         </property>
         <property name="text">
          <string>SHOPPING LIST</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(168, 165, 173);
font: bold 12px; color: #1c1b1b;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="swapButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: bold 14px;
    color: white;</string>
         </property>
         <property name="text">
          <string>LOW ON:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(168, 165, 173);
font: bold 12px; color: white;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(168, 165, 173);
font: bold 12px; color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">font: bold 14px;
    color: white;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">font: bold 14px;
    color: white;</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: bold 14px;
    color: white;</string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ShoppingPage1</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
